lib/jekyll-feed.rb

```diff
@@ -69,27 +67,13 @@ def source_path
       File.expand_path "feed.xml", File.dirname(__FILE__)
     end
 
-    # Destination for feed.xml file within the site source directory
-    def destination_path
-      if @site.respond_to?(:in_dest_dir)
-        @site.in_dest_dir(path)
-      else
-        Jekyll.sanitized_path(@site.dest, path)
-      end
-    end
-
-    # copy feed template from source to destination
-    def write
-      FileUtils.mkdir_p File.dirname(destination_path)
-      File.open(destination_path, 'w') { |f| f.write(feed_content) }
-    end
-
     def feed_content
-      site_map = PageWithoutAFile.new(@site, File.dirname(__FILE__), "", path)
-      site_map.content = File.read(source_path).gsub(/\s*\n\s*/, "\n").gsub(/\n{%/, "{%")
-      site_map.data["layout"] = nil
-      site_map.render(Hash.new, @site.site_payload)
-      site_map.output
+      feed = PageWithoutAFile.new(@site, File.dirname(__FILE__), "", path)
+      feed.content = File.read(source_path).gsub(/\s*\n\s*/, "\n").gsub(/\s+{%/, "{%").gsub(/\s+</,"<")
+      feed.data["layout"] = nil
+      feed.data['sitemap'] = false
```

Comment:
Would you mind unifying the string literals? Probably go with double quotes as they're used above.